tests/test-media-item-mutations.php

```diff
@@ -0,0 +1,803 @@
+			'post_author' => $this->admin,
+		] );
+		wp_set_current_user( $this->author );
+		$this->create_variables['input']['parentId'] = $post;
+		$actual = $this->createMediaItemMutation();
+		$this->assertArrayHasKey( 'errors', $actual );
+		$this->create_variables['input']['parentId'] = $this->parentId;
+	}
+
+	/**
+	 * This function tests the createMediaItem mutation
+	 *
+	 * @source wp-content/plugins/wp-graphql/src/Type/MediaItem/Mutation/MediaItemCreate.php
+	 * @access public
+	 * @return void
+	 */
+	public function testCreateMediaItemMutation() {
+
+		/**
+		 * Set the current user as the admin role so we
+		 * can properly test the mutation
+		 */
+		wp_set_current_user( $this->admin );
+
+		/**
+		 * Create the createMediaItem
+		 */
+		$actual = $this->createMediaItemMutation();
+
+		$media_item_id = $actual["data"]["createMediaItem"]["mediaItem"]["id"];
+		$attachment_id = $actual["data"]["createMediaItem"]["mediaItem"]["mediaItemId"];
+
+		$expected = [
+			'data' => [
+				'createMediaItem' => [
+					'clientMutationId' => $this->clientMutationId,
+					'mediaItem' => [
+						'id'               => $media_item_id,
+						'mediaItemId'      => $attachment_id,
+						'title'            => $this->title,
+						'description'      => apply_filters( 'the_content', $this->description ),
+						'altText'          => $this->altText,
+						'caption'          => apply_filters( 'the_content', $this->caption ),
+						'commentStatus'    => $this->commentStatus,
+						'date'             => $this->date,
+						'dateGmt'          => $this->dateGmt,
+						'slug'             => $this->slug,
+						'status'           => strtolower( $this->status ),
+						'pingStatus'       => $this->pingStatus,
+						'mimeType'         => 'image/gif',
+					],
+				],
+			],
+		];
+
+		$this->assertEquals( $expected, $actual );
+
+	}
+
+	/**
+	 * Update a media item using the updateMediaItem mutation
+	 *
+	 * @access public
+	 * @return array $actual
+	 */
+	public function updateMediaItemMutation() {
+
+		/**
+		 * Prepare the updateMediaItem mutation
+		 */
+		$mutation = '
+		mutation updateMediaItem( $input: updateMediaItemInput! ){
+		  updateMediaItem (input: $input){
+		    clientMutationId
+		    mediaItem {
+		      id
+		      mediaItemId
+		      date
+		      dateGmt
+		      slug
+		      status
+		      title
+		      commentStatus
+		      pingStatus
+		      altText
+		      caption
+		      description
+		      mimeType
+		      author {
+		        id
+		      }
+		    }
+		  }
+		}
+		';
+
+		$actual = do_graphql_request( $mutation, 'updateMediaItem', $this->update_variables );
+
+		return $actual;
+	}
+
+	/**
+	 * Execute the request with a fake mediaItem id. An error
+	 * should occur because we didn't pass the id of the mediaItem we
+	 * wanted to update.
+	 *
+	 * @source wp-content/plugins/wp-graphql/src/Type/MediaItem/MediaItemUpdate.php:57
+	 * @access public
+	 * @return void
+	 */
+	public function testUmiInvalidId() {
```

Comment:
let's update the test names to be more explicit. 

`testUmi` doesn't mean anything to me if I'm working on an unrelated feature and see that my code broke `testUmi`. . .but if I see that I broke `testUpdateMediaItemMutationInvalidID` then I have more clarity and don't have to waste time trying to find out what `Umi` stands for

same with `Cmi` and `Dmi`. . .let's be explicit and use `CreateMediaItem` and `DeleteMediaItem`

A quick search/replace should work